docs/geedocs/5.2.3/answer/7160003.html

```diff
@@ -141,6 +141,12 @@ <h5>
               This allows upgrading Fusion on machines where changing asset root ownership recursively would be very expensive.
             </td>
           </tr>
+          <tr>
+            <td>913</td>
```

Comment:
the fixed issue is 912 (https://github.com/google/earthenterprise/issues/912). I would change `913` to `912`

Reply:
Oops, grabbed the PR number instead of the issue number...